### DISCUSSION TITLE
Can we configure installation per-user or per-machine for bundle and package in BootstrapApplication?

### DISCUSSION THREAD

**Discussion by Federau** [wixtoolset/issues#7782] [Questions] [answered]
Hey guys. I am using Wix4.0.1 to build a bundle which contains two packages, Can I configure installation per-user or per-machine for bundle and package in BootstrapApplication?

**Comment by robmen** [comment, marked as answer]
Yes, but you have to pick per-user or per-machine when building the bundle.

**Reply by Federau** [reply]
but when I add a dual-purpose msi in a bundle, I got an wix error 
WIX1133: Bundles require a package to be either per-machine or per-user.The MSI 'WebViewSetup' ALLUSERS Property is set to '2' which may change from per-user to per-machine at install time. The Bundle will assume the package is per-machine and will not work correctly if that changes. If possible, remove the Property with Id='ALLUSERS' and use Package/@InstallScope attribute instead.

**Reply by robmen** [reply]
That's not an error. It's a warning.

**Reply by Federau** [reply]
but when I use visual studio 2022 to build it, it shows it is an error and cannot get bundle exe file in output directory.

**Reply by robmen** [reply]
You probably set warnings as errors.

**Reply by Federau** [reply]
I cannot find where I set warnings as errors, but using suppress warning works. BTY, how to set msi package scope to perUser when building the bundle?  I noticed that in wix3 we have InstallScope attribute under MsiPackage, but only a ForcePerMachine in wix4.
